Dockerfile

```diff
@@ -2,22 +2,31 @@
 # https://hub.docker.com/layers/nginxinc/nginx-unprivileged/1.24.0-bullseye/images/sha256-a8ec652916ce1e7ab2ab624fe59bb8dfc16a018fd489c6fb979fe35c5dd3ec50
 FROM docker.io/nginxinc/nginx-unprivileged:1.24.0-bullseye@sha256:ac0654a834233f7cc95b3a61550a07636299ce9020b5a11a04890b77b6917dc4
 
-LABEL org.opencontainers.image.title frames
```

Comment:
this might need to be reverted back to using the `=` and the `""` and same with the following line